### PR TITLE
Feat/documentation of permissions

### LINKED ISSUE #64: Documentation of permissions
### Reason

Create documentation of permissions

### Description

Permission documentation

### Success Criteria

- [ ] Docs Created

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by JesseMonteiro**
close #64

### CODE REVIEW

File: TikiClient/Classes/TikiClient.swift [hunk 1]
```diff
@@ -14,12 +14,14 @@ import SwiftUI
...
 /// that call the underlying libraries to perform common operations. Programmers can use it to
 /// simplify the integration process or opt for individual libraries based on their specific needs.
+
 public class TikiClient {
     
     public static let auth = AuthService()
     public static let capture = CaptureService()
     public static let license = License()
     public static let tracking = Tracking()
+    public static let location =  ()
```
Comment (ricardobrg) [inline TikiClient/Classes/TikiClient.swift:24]: This should not be here

File: TikiClient/Classes/TikiClient.swift [hunk 2]
```diff
@@ -31,7 +33,8 @@ public class TikiClient {
     ///   - company: The legal information of the company.
     public static func initialize(userId: String, completion: @escaping (String?) -> Void) {
         if(config == nil){
-            fatalError("Config is nil")
+            completion("Config is nil")
```
Comment (ricardobrg) [inline TikiClient/Classes/TikiClient.swift:36]: It shoudl throw an error or call a onError callback

**Review by ricardobrg — CHANGES_REQUESTED** [review]

File: TikiClient/Classes/TikiClient.swift (hunk 1 above)
Comment (JesseMonteiro) [reply TikiClient/Classes/TikiClient.swift:24]: Was an mistake, i erase the reference without noticing.

File: TikiClient/Classes/TikiClient.swift (hunk 2 above)
Comment (JesseMonteiro) [reply TikiClient/Classes/TikiClient.swift:36]: done

**Review by ricardobrg — APPROVED** [review]

**Pull request merged**